lib/pronto/formatter/pull_request_formatter.rb

```diff
@@ -9,6 +9,7 @@ def existing_comments(_, client, repo)
 
       def submit_comments(client, comments)
         comments.each { |comment| client.create_pull_comment(comment) }
+        # TODO: This is here why? It looks like Github specific stuffs
       rescue Octokit::UnprocessableEntity, HTTParty::Error => e
```

Comment:
The gitlab gem has it's own [error class](https://github.com/NARKOZ/gitlab/blob/master/lib/gitlab/error.rb) and uses HTTParty as well, so changing the first rescued class should be enough.